### PR TITLE
Remove Tail header and L4 dest port option

### PR DESCRIPTION
**Pull request by jklr**
- Remove L4 dest port option for INT over TCP/UDP encap
- Remove Tail header from INT over TCP/UDP encap
- Fix section references
- Use the same look and feel with p4-16 spec

### CODE REVIEW

File: telemetry/specs/INT.mdk [hunk 1]
```diff
@@ -419,11 +519,12 @@ INT sink either do not parse beyond layer-4 headers or can skip through the
...
+
+Note that INT over UDP can be used even when the packet is encapsulated by VXLAN,
 Geneve, or GUE (Generic UDP Encapsulation). INT over TCP/UDP also makes it
 easier to add INT stack into outer, inner, or even both layers. In such cases
 both INT header stacks carry information for respective layers and need not be
-considered interfering with each other.)
+considered interfering with each other.
 
 A field in Ethernet, IP, or TCP/UDP should indicate if the
 INT header exists after the TCP/UDP header. We propose three options.
```
Comment (mickeyspiegel) [inline telemetry/specs/INT.mdk]: s/three options/two options

File: telemetry/specs/INT.mdk [hunk 2]
```diff
@@ -2030,3 +2098,7 @@ assumptions in section [#sec-examples]
     against PSA architecture instead of v1model.
 * 2018-04-03
   - Some more editorial changes for v1.0
+* 2018-04-10
+  - Removed the option to modify L4 destination port to indicate INT over
+    TCP/UDP.
+  - Removed INT Tail header from INT over TCP/UDP encapsulation.
```
Comment (mickeyspiegel) [inline telemetry/specs/INT.mdk:2104]: - Added DSCP to the INT over TCP/UDP shim header

**Review by mickeyspiegel — APPROVED** [review]

File: telemetry/specs/INT.mdk (hunk 1 above)
Comment (jklr) [reply telemetry/specs/INT.mdk]: Done

File: telemetry/specs/INT.mdk (hunk 2 above)
Comment (jklr) [reply telemetry/specs/INT.mdk:2104]: Done

**Review by mhira1 — APPROVED** [review]

**Pull request merged**